core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GoogleServicesDAO.scala

```diff
@@ -172,11 +172,39 @@ abstract class GoogleServicesDAO(groupsPrefix: String) extends ErrorReportable {
     */
   def cleanupDMProject(projectName: RawlsBillingProjectName): Future[Unit]
 
+  /**
+    * Removes the IAM policies to the project's existing policies
+    * @return true if the policy was actually changed
+    */
+  def removePolicyBindings(projectName: RawlsBillingProjectName, policiesToRemove: Map[String, Set[String]]): Future[Boolean] = updatePolicyBindings(projectName) { existingPolicies =>
```

Comment:
`updatePolicyBindings` is a function that takes a function with a single parameter as an argument. The `=>` notation is saying that this is a function, the things on the left are the names of the parameters (just 1 in this case, `existingPolicies`) and the stuff on the right is the body of the function. The type of the parameter is not explicitly specified because `updatePolicyBindings` says what it must be.